expression/schema.go

```diff
@@ -31,8 +31,9 @@ func (ki KeyInfo) Clone() KeyInfo {
 
 // Schema stands for the row schema and unique key information get from input.
 type Schema struct {
-	Columns []*Column
-	Keys    []KeyInfo
+	Columns    []*Column
+	Keys       []KeyInfo
```

Comment:
It will be better to add some comments to tell others what's the difference between `Keys` and `UniqueKeys`

Reply:
done